web/about_test.go

```diff
@@ -30,7 +29,7 @@ func TestAboutHandlerPremium(t *testing.T) {
 	}
 
 	resp := httptest.NewRecorder()
-	req, err := http.NewRequest("GET", "/about", nil)
+	req := httptest.NewRequest("GET", "/about", nil)
```

Comment:
@stefanotorresi I think that the brute `copy/paste` is not a good idea hehe
Here for example, we would need to remove the next `if err != nil` condition. That it doesn't break any test is a coincidence, as we create the app before getting the `err` variable

Reply:
gahh, you're right. I couldn't do that with regexes 😁